public/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css">
```

Comment:
bulma already present in node modules, you can just set it in css file:
`@import "bulma/css/bulma.css";`

Reply:
link is also a cdn